CAP/gap/CategoryMorphisms.gd

```diff
@@ -65,6 +65,137 @@ DeclareGlobalFunction( "INSTALL_TODO_LIST_FOR_EQUAL_MORPHISMS" );
 
 DeclareGlobalVariable( "PROPAGATION_LIST_FOR_EQUAL_MORPHISMS" );
 
+###################################
+##
+#! @Section Random Morphisms
+##
+###################################
+
+#! @Description
+#! The arguments are two objects $a$ and $b$ in a category $C$ and an integer $n$.
+#! The output is a random morphism $\alpha: a \rightarrow b$ in $C$.
+#! The integer $n$ indicates the degree of complexity of the created morphism
+#! $\alpha$ and its interpretation differs from one category to another.
+#! For example in the category of rational vector spaces, $n$ can indicate that 
+#! all entries of the matrix of $f$ are less or equal to $n$.
+#! @Returns a morphism in $\mathrm{Hom}(a,b)$
```

Comment:
This description needs to take into account that the output might be `fail`.
The same holds for the other descriptions.

Reply:
https://github.com/homalg-project/CAP_project/pull/239/commits/e08e47f6cbb702462cc95e6452f93c3db1283499